parsers/AU.py

```diff
@@ -321,6 +330,7 @@ def fetch_production(country_code=None, session=None):
     df = pd.read_csv(url)
     data = {
         'countryCode': country_code,
+        'datetime': arrow.get('2001-01-01T00:00:00+1000').datetime,
```

Comment:
It'd never get through unless there was no data in the upstream API at all, and at that point your backend would probably ignore it anyway? But true, I was trying to be a bit too clever, rewrote it.